weeder.toml

```
@@ -3,5 +3,6 @@ roots = [
   "^Main.main$",
   "^Paths_",
   "^Yesod.Session.Persist.Yesod.makeSessionBackend$",
+  "^Yesod.Session.Memcache.Yesod.makeSessionBackend$",
   "^Yesod.Session.Options.hoistOptions$",
```

```suggestion
  "^Yesod.Session.Memcache.Yesod.makeSessionBackend$",
  "^Yesod.Session.Options.hoistOptions$",
  "^Yesod.Session.Persist.Yesod.makeSessionBackend$",
```

Maybe fix the sorting while here?